src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -166,10 +167,10 @@ export class AttachedAccountsTreeItem extends RootTreeItem<ISubscriptionRoot> {
             if (port) {
                 if (defaultExperience.api === API.MongoDB) {
                     // Mongo shell doesn't parse passwords with slashes, so we need to URI encode it
-                    connectionString = `mongodb://localhost:${encodeURIComponent('C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==')}@localhost:${port}/?ssl=true`;
+                    connectionString = `mongodb://localhost:${encodeURIComponent(ext.emulatorPassword)}@localhost:${port}/?ssl=true`;
```

Comment:
How did Stephen get the bug to occur if we're using `encodeURIComponent` here? Did he manually attach to the local emulator? We should handle the manual attach scenario anyways - I'm just curious.

Reply:
The bug happened outside the attach emulator code. Most likely the emulator was already attached, before encodeURI was used in the code. 
Tested with master, and a newly attached emulator doesn't repro the bug because the connectionString is now correct.

Reply:
This happened because I rolled back the fix to get another bug to repro. Then the bad URI got saved in settings.